CMakeLists.txt

```diff
@@ -18,23 +18,16 @@ if(CONFIG_BOARD_NRF52_BSIM)
  https://babblesim.github.io/folder_structure_and_env.html")
   endif()
 
-  # These include directories must be added before those from the hal_nordic
-  # module, so that the local modified versions of nrfx files are used instead
-  # of those from the original nrfx.
-  target_include_directories(zephyr_interface BEFORE INTERFACE
-    ${CMAKE_CURRENT_SOURCE_DIR}/src/
```

Comment:
How do you ensure the headers provided here are included instead of the ones from the nRFx west module? (both provide nrfx_config.h and nrfx_glue.h)

Reply:
I did so by not adding that particular folder to the folders to be included. To safeguard this, I've now added a safeguard( `#ifdef __ZEPHYR__` ) in these.